sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/implementation/compatibility/AzureSpringBootVersionVerifier.java

```diff
@@ -68,19 +68,20 @@ public VerificationResult verify() {
     private String errorDescription() {
         String versionFromManifest = this.getVersionFromManifest();
         return StringUtils.hasText(versionFromManifest) ? String.format("Spring Boot [%s] is not compatible with this"
-            + " Spring Cloud Azure release train", versionFromManifest) : "Spring Boot is not compatible with this "
-            + "Spring Cloud Azure release train";
+            + " Spring Cloud Azure release", versionFromManifest) : "Spring Boot is not compatible with this "
+            + "Spring Cloud Azure release";
     }
 
     private String action() {
-        return String.format("Change Spring Boot version to one of the following versions %s .%n"
+        return String.format("Change Spring Boot version to one of the following versions %s.%n"
                 + "You can find the latest Spring Boot versions here [%s]. %n"
-                + "If you want to learn more about the Spring Cloud Azure Release train compatibility, "
-                + "you can visit this page [%s] and check the [Release Trains] section.%nIf you want to disable this "
-                + "check, just set the property [spring.cloud.azure.compatibility-verifier.enabled=false]",
+                + "If you want to learn more about the Spring Cloud Azure compatibility, "
+                + "you can visit this page [%s] and check the [Which Version of Spring Cloud Azure Should I Use] section.%n"
```

Comment:
Hi @chenrujun , I only update the [reference link](https://github.com/Azure/azure-sdk-for-java/wiki/Spring-Versions-Mapping#which-version-of-spring-cloud-azure-should-i-use) available, please note we need to make sure the section title doesn't change.

Reply:
@moarychan , if you have this requirement, it's better to written it in that wiki page because I'm not the only person who is possible to update that page.